scripts/migrate.sh

```
@@ -0,0 +1,61 @@
+
+# Function to apply migration via MCP
+apply_via_mcp() {
+    local migration_file=$1
+    local migration_name=$(basename "$migration_file" .sql)
+
+    echo "📝 Applying $migration_name via Supabase MCP..."
+
+    # Use the Supabase MCP tool to apply the migration
+    # This bypasses CLI connectivity issues
+    doppler run -- pnpm exec supabase-mcp apply-migration "$migration_file"
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)